Runtime/Scripts/Extensions/MPEG/Interactivity/UserInputSceneTrigger.cs

```diff
@@ -114,7 +124,6 @@ public static string GetBindingFromUserInputDescription(string description)
                 case "touchscreen":
                     builder.Append("<Touchscreen>");
```

Comment:
Is this covered by a specification ?

Reply:
No, this is to work with Unity input path system, and it need to be updated to support an openXR path

Reply:
No, this is to work with Unity input path system, and it need to be updated to support an openXR path